design/FY2017/NFP-Enhancement for Urban Microclimate.md

```diff
@@ -0,0 +1,99 @@
+Enhancement for Simulation using Urban Microclimate Conditions
+================
+**Tianzhen Hong, Xuan Luo, LBNL**
+**Original: June 27, 2017**
+
+# Justification for New Feature 
+
+With the global trend towards urbanization, energy use and the associated GHG emissions are growing significantly in cities. Reducing energy use in the building sector is a key strategy for cities to achieve their energy and environmental goals. The urban climate and microclimate can strongly affect the building energy demand. As air temperatures in urban areas are higher due to the urban heat island (UHI) effect and the wind speeds can be lower due to wind sheltering, the microclimate in urban areas differs significantly from the climate in rural areas. The proposed enhancement intends to enable EnergyPlus simulations to use the local ambient conditions, which can be from measurements or CFD simulation, including parameters of outdoor air velocity and direction, dry-bulb temperature and humidity.  
+
+# Overview 
+Urban energy modeling is an emerging field that seeks to model a group of buildings and the surrounding urban micro-climate. The microclimate around a building, established through the interactions with other buildings or the natural environment, is a significant factor influencing building energy consumption. On the one hand, the energy consumption of urban buildings is affected by the surrounding microclimate which differs from standard weather data, and the mutual obstructions between buildings which decrease sunlight and wind potentials for internal solar gains and passive cooling. On the other hand, the building construction itself affects both outdoor and indoor microclimate. As Figure 1 shows, the building model in EnergyPlus serves as the boundary condition in the urban atmosphere model, and the building exchanges mass (air flow) and heat with the surrounding environment, including exhaust air from fans, DX condensing units, cooling towers, boilers, etc. 
+
+![EplusUrban](EnergyPlusUrbanClimate.png)
+
+***Figure 1 EnergyPlus data exchange with urban microclimate model***
+
+The current implementation of EnergyPlus assumes a default vertical temperature gradient. It does not provide a direct link to allow data exchange with urban microclimate models to override local environmental data at the zone and surface levels, such as the outdoor air temperature, humidity, wind velocity and direction. However, there are reportable surface and zone outdoor conditions tracked separately as:
```

Comment:
Why is this problem being addressed with EMS sensors only (which can be actuated to include input from external preprocessed CFD simulations) but the external shading and LWR proposals have input objects which reference large amounts of data (e.g. CSV files with shading coefficients and surface temperatures).  This seems like it sets up two different paradigms (large amounts of input data vs run time coupling via EMS actuators) for coupling E+ with other calculation engines (shading or CFD).  What is the rationale for the difference in approach?  Is the idea that CFD is more coupled to the building model and thus requires co-simulation?  What about LWR, isn't that coupled as well?